Exercise_A/index.html

```diff
@@ -10,7 +10,9 @@
   </head>
   <body>
     <!-- CODE HERE -->
-
+<a href="about.html">
+  <img src="Ash_Pikachu.png" alt="pic of pikachu">
```

Comment:
You did a good job adding an anchor tag and linking it to another HTML page. Well done!